packages/cohttp-lwt/cohttp-lwt.6.0.0~alpha2/opam

```diff
@@ -0,0 +1,63 @@
+opam-version: "2.0"
+synopsis: "CoHTTP implementation using the Lwt concurrency library"
+description: """
+This is a portable implementation of HTTP that uses the Lwt concurrency library
+to multiplex IO.  It implements as much of the logic in an OS-independent way
+as possible, so that more specialised modules can be tailored for different
+targets.  For example, you can install `cohttp-lwt-unix` or `cohttp-lwt-jsoo`
+for a Unix or JavaScript backend, or `cohttp-mirage` for the MirageOS unikernel
+version of the library. All of these implementations share the same IO logic
+from this module."""
+maintainer: ["Anil Madhavapeddy <anil@recoil.org>"]
+authors: [
+  "Anil Madhavapeddy"
+  "Stefano Zacchiroli"
+  "David Sheets"
+  "Thomas Gazagnaire"
+  "David Scott"
+  "Rudi Grinberg"
+  "Andy Ray"
+  "Anurag Soni"
+]
+license: "ISC"
+homepage: "https://github.com/mirage/ocaml-cohttp"
+doc: "https://mirage.github.io/ocaml-cohttp/"
+bug-reports: "https://github.com/mirage/ocaml-cohttp/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "ocaml" {>= "4.08"}
+  "http" {= version}
+  "cohttp" {= version}
+  "lwt" {>= "2.5.0"}
```

Comment:
```suggestion
  "lwt" {>= "5.4.0"}
```
```
#=== ERROR while compiling cohttp-lwt.6.0.0~alpha2 ============================#
# context              2.2.0~alpha2~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | file:///home/opam/opam-repository
# path                 ~/.opam/4.08/.opam-switch/build/cohttp-lwt.6.0.0~alpha2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p cohttp-lwt -j 255 @install
# exit-code            1
# env-file             ~/.opam/log/cohttp-lwt-7-6d1634.env
# output-file          ~/.opam/log/cohttp-lwt-7-6d1634.out
### output ###
# File "/home/opam/.opam/4.08/lib/ocaml-compiler-libs/bytecomp/ocaml-compiler-libs.bytecomp.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ocaml-compiler-libs/common/ocaml-compiler-libs.common.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ocaml-compiler-libs/shadow/ocaml-compiler-libs.shadow.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ocaml-compiler-libs/toplevel/ocaml-compiler-libs.toplevel.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ppx_derivers/ppx_derivers.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I cohttp-lwt/src/.cohttp_lwt.objs/byte -I /home/opam/.opam/4.08/lib/base64 -I /home/opam/.opam/4.08/lib/cohttp -I /home/opam/.opam/4.08/lib/http -I /home/opam/.opam/4.08/lib/http/__private__/http_bytebuffer/.public_cmi -I /home/opam/.opam/4.08/lib/logs -I /home/opam/.opam/4.08/lib/lwt -I /home/opam/.opam/4.08/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.08/lib/re -I /home/opam/.opam/4.08/lib/re/posix -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/sexplib0 -I /home/opam/.opam/4.08/lib/stringext -I /home/opam/.opam/4.08/lib/uri -I /home/opam/.opam/4.08/lib/uri-sexp -no-alias-deps -open Cohttp_lwt__ -o cohttp-lwt/src/.cohttp_lwt.objs/byte/cohttp_lwt__Connection.cmo -c -impl cohttp-lwt/src/connection.pp.ml)
# File "cohttp-lwt/src/connection.ml", line 291, characters 8-21:
# 291 |         Lwt.dont_wait (fun () -> reader connection) on_failure;
#               ^^^^^^^^^^^^^
# Error: Unbound value Lwt.dont_wait
```

Reply:
Thanks